CONTRIBUTION.md

```diff
@@ -1,94 +1,136 @@
 # How to contribute
 
 Windows Machine Config Operator is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This
-document outlines some of the conventions on commit message formatting and other resources to help get contributions into the project.  
+document outlines some of the conventions on commit message formatting and other resources to help get
+contributions into the project.  
 
+## New Contributor guide
 
-## Reporting bugs and creating issues
+To get an overview of what the WMCO is and does, please take a look at the [README](README.md) first. In order to
```

Comment:
For readability, I would change to:
To get an overview of what the WMCO does, please take a look at the README.